sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Ucfg/UcfgBuilder_Instructions.cs

```diff
@@ -196,6 +210,31 @@ public string Foo(string s)
                 );
         }
 
+        [TestMethod]
+        public void Assignments_AutoProperties()
+        {
+            const string code = @"
+namespace Namespace
+{
+    public class Class1
+    {
+        private string Property { get; }
+
+        public Class1(string s)
+        {
+            Property = s;   // ignored, property has no SetMethod
+                            // This should be ok for now because we don't really care about
+                            // properties without accessors, because they behave as pure fields.
```

Comment:
This used to generate __unknown instruction, but it broke the string argument check that I added